load.php

```diff
@@ -104,6 +104,7 @@ function get_config() {
 		'seo'              => false,
 		'redirects'        => false,
 		'bylines'          => false,
+		'hm-stack'         => true,
```

Comment:
I'd say we should call this `hm-stack-api` maybe, given the uses elsewhere.

Reply:
I actually realized that we shouldn't be giving people the opportunity to turn this off so moved the loading so a `require` along with the platform-ui plugin.

Reply:
Hmm i think it's possible we would - so we could disable the UI if wanted, this would allow us to merge this into master without rolling out to all clients at once.